### PR TITLE
Translate ObjectReference #32433

### PR DESCRIPTION
**Pull request by xuezhixin**
Translate ObjectReference https://github.com/kubernetes/website/issues/32433

### CODE REVIEW

**Comment by xuezhixin** [comment]
https://github.com/kubernetes/website/pull/32447#issuecomment-1076369962****

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/api/core/v1"
+  kind: "ObjectReference"
+content_type: "api_reference"
+description: "ObjectReference 包含足够的信息，可以让您检查或修改引用的对象。"
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:7]: Please attach the English content for the frontmatter, as comment after line 12.

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+`import "k8s.io/api/core/v1"`
+
+<!--
+ObjectReference contains enough information to let you inspect or modify the referred object.
+
+<hr>
+-->
+
+ObjectReference包含足够的信息，允许您检查或修改引用的对象。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:34]: 全文不用 '您'，用'你'.
inspect -> 检视

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  Specific resourceVersion to which this reference is made, if any. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency
+
+- **uid** (string)
+
+  UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids
+-->
+
+- **apiVersion** (string)
+
+  引用的API版本。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:71]:
```suggestion
  被引用者的API版本。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+- **fieldPath** (string)
+
+  如果引用的是对象的一部分而不是整个对象，则该字符串应包含有效的JSON/Go字段访问语句，如desiredState.manifest.containers[2]。例如，如果对象引用是pod中的一个容器，那么它将采用如下值：“spec.containers{name}”（其中“name”指触发事件的容器的名称），或者如果没有指定容器名称，“spec.containers[2]”（此pod中会索引2的容器）。选择这种语法只是为了有一些定义良好的方法来引用对象的一部分。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:75]: 注意中英文间留白

```suggestion
  如果引用的是对象的一部分而不是整个对象，则该字符串应包含有效的 JSON/Go 字段访问语句，
  如 `desiredState.manifest.containers[2]`。例如，如果对象引用针对的是 Pod中的一个容器，
  此字段取值类似于：`spec.containers{name}`（其中 `name` 指触发事件的容器的名称），
  或者如果没有指定容器名称，`spec.containers[2]`（此 Pod 中索引为 2 的容器）。
  选择这种语法只是为了有一些定义良好的方法来引用对象的一部分。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+- **kind** (string)
+
+  有点指代物。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:79]:
```suggestion
  被引用者的类别（kind）。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+- **name** (string)
+
+  所指对象的名称。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:83]:
```suggestion
  被引用对象的名称。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+- **namespace** (string)
+
+  引用对象的命名空间。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:87]:
```suggestion
  被引用对象的名字空间。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+- **resourceVersion** (string)
+
+  此引用的特定资源版本（如果有）。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:91]:
```suggestion
  被引用对象的特定资源版本（如果有）。更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+- **uid** (string)
+
+  引用对象的UID。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/object-reference.md:95]:
```suggestion
  被引用对象的 UID。更多信息：https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids
```

**Comment by xuezhixin** [comment]
Thanks for correcting .

**Pull request merged**